flashinfer/comm/torch_symmetric_memory.py

```
@@ -0,0 +1,76 @@
+        return
+    _compat_patched = True
+
+    import torch.distributed as dist
+
+    _original_destroy = dist.destroy_process_group
+
+    @functools.wraps(_original_destroy)
+    def _patched_destroy(group=None):
+        saved_count = c10d._world.group_count
+        _original_destroy(group)
+        # WORLD destruction resets group_count to 0 – restore it so the next
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)